wp-content/themes/wp-starter/blocks/alert-banner/block.php

```diff
@@ -10,17 +10,15 @@
  *
  * @return void
  */
-function alert_banner_dismiss_button(): void {
+function alert_banner_dismiss_button($id = 'alert-banner'): void {
 	$dismiss_button = sprintf(
-		'<button class="dismiss" aria-label="%s" title="%s">',
+		'<button class="dismiss" aria-label="%s" title="%s" @click="%s = !%s">',
 		esc_attr__( 'Dismiss alert banner', 'wp-starter' ),
-		esc_attr__( 'Dismiss', 'wp-starter' )
+		esc_attr__( 'Dismiss', 'wp-starter' ),
+		$id,
+		$id
 	);
-	$dismiss_button .= '<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24" fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round" stroke-linejoin="round">';
-	$dismiss_button .= '<line x1="18" y1="6" x2="6" y2="18"></line>';
-	$dismiss_button .= '<line x1="6" y1="6" x2="18" y2="18"></line>';
-	$dismiss_button .= '</svg>';
 	$dismiss_button .= '</button>';
 
-	echo wp_kses_post( $dismiss_button );
+	echo ( $dismiss_button );
```

Comment:
Had to remove `wp_kses_post` because it was removing all the Alpine code. We may need to add a filter to `wp_kses_post` to allow the Alpine code.

Reply:
Nah, I think it's fine. Just need to remove the parenthesis.